### PR TITLE
dev: hello.py trying creating PR

### PR DESCRIPTION
**Pull request by retr0k89**
Just a simple test on availability on creating PR

### CODE REVIEW

File: hello.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+def hello():
+    print("Hello PyCon APAC 2022")
+
+if __name__ == '__main__':
+    hello()
```
Comment (tai271828) [inline hello.py:5]: Hi @adinpearce , maybe time you want to pay attention to the end of line.

Comment (retr0k89) [reply hello.py:5]: Sorry, I will need more specific instructions.

File: hello.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+def hello():
```
Comment (tai271828) [inline hello.py:1]: You may probably need shebang/hashbang.

File: hello.py (hunk 1 above)
Comment (tai271828) [reply hello.py:5]: Selection_017

Comment (tai271828) [reply hello.py:5]: It's ok. I fixed that today along with michelle! https://github.com/yuyinwangw/cli_test/pull/2/files#diff-f10daf8f1ee9ded24fd9ddd7ca6a8171bb0888894d9b805a75dac52078bd4cfcL5

**Pull request merged**